src/App.js

```diff
@@ -1,12 +1,75 @@
-import logo from './logo.svg';
 import './App.css';
+import {useEffect, useState} from 'react';
+import './disign.css';
+import ObjectInfo from "./components/ObjectInfo";
 
 function App() {
-  return (
-    <div className="App">
-        <div> hello </div>
+
+    const [leftarr, setlarr] = useState([]);
+    const [rightarr, setrarr] = useState([]);
+    const [ backgroundColor, setBackgroundColor ] = useState('#50e3e3');
+
+    useEffect(() => {
+        const dataJSON = require('./MOCK_DATA.json');
+        setlarr(dataJSON);
+        console.log(leftarr.length);
```

Comment:
remove

Reply:
Whyshould i remove it? This is for adding some data to page